client/container_resize.go

```diff
@@ -19,9 +19,10 @@ func (cli *Client) ContainerExecResize(ctx context.Context, execID string, optio
 }
 
 func (cli *Client) resize(ctx context.Context, basePath string, height, width uint) error {
+	// FIXME(thaJeztah): the API / backend accepts uint32, but container.ResizeOptions uses uint (uint64)
```

Comment:
```suggestion
	// FIXME(thaJeztah): the API / backend accepts uint32, but container.ResizeOptions uses uint
```

uint is a separate type from uint64 - it's **at least** uint32, but not strictly an alias to uint64 or uint32

Reply:
Ah, good point; I'll update